src/WebhookClientServiceProvider.php

```diff
@@ -44,9 +48,13 @@ public function packageBooted()
         $this->app->bind(WebhookConfig::class, function () {
             $routeName = Route::currentRouteName() ?? '';
 
-            $routeNameSuffix = Str::after($routeName, 'webhook-client-');
+            $configName = Str::after($routeName, 'webhook-client-');
 
-            $configName = Str::before($routeNameSuffix, '.');
+            if (config('webhook-client.add_unique_token_to_route_name', false)) {
```

Comment:
@it-can I think this should probably default to true since the shared route names already break route caching and there is a documented feature that causes that scenario. I can definitely see the benefit of calling the routes by name but route naming is more of an underlying structure of the package and not a documented feature.

Reply:
@freekmurze can decide on this one too...

Reply:
I think it should be set to `false` for now, so we don't break stuff for existing users.

For a next major version, I'll consider setting it to `true` by default.